.github/workflows/CI.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           path: ~/.julia/artifacts
           key: ${{ runner.os }}-test-artifacts-${{ hashFiles('**/Project.toml') }}
           restore-keys: ${{ runner.os }}-test-artifacts
+      - run: julia --project -e 'using Pkg; Pkg.develop([PackageSpec(; path="OndaEDFSchemas.jl")])'
```

Comment:
Should only be needed until OndaEDFSchemas.jl is registered. My personal preference would be to have this work split into two PRs. Also, I do find using a committed Manifest.toml is a cleaner way to do that but mainly that's only useful while developing the PR

Reply:
I _think_ it is registered at this point.  Without this the registered version is picked up, which means that the OndaEDF CI will not pick up the OndaEDFSchemas changes in a PR.